Sources/NIOSSH/Constants.swift

```diff
@@ -12,6 +12,10 @@
 //
 //===----------------------------------------------------------------------===//
 
-enum Constants {
+public enum Constants {
```

Comment:
Not sure this is the best place, maybe we need to type `Defaults` or smth?

Reply:
I agree with you, I think `Defaults` would be better.

Reply:
Decided to change this a bit, I will rename constant to `bundledTransportProtectionSchemes`, I quite like this idea from [Joannis](https://github.com/Joannis)'s PR, wdyt?

Reply:
LGTM.